components/Calendar/Calendar.js

```diff
@@ -0,0 +1,142 @@
+// #FFFFFF - white
+// #F5F5F5 - off-white
+// #B0B4D4 - light purp
+// #84C7D0 - sea blue
+// #4BD2D2 - ugly blue
+
+
+import { useState, useEffect } from 'react';
+// import { Scheduler, WeekView, DayView, Appointments, Toolbar, DateNavigator, ViewSwitcher,}
+//import moment from 'moment';
+import { styled, alpha } from '@mui/material/styles';
+import Paper from '@mui/material/Paper';
+import { ViewState } from '@devexpress/dx-react-scheduler';
+import {
+  Scheduler,
+  WeekView,
+  Toolbar,
+  DateNavigator,
+  Appointments,
+  TodayButton,
+  AppointmentTooltip,
+} from '@devexpress/dx-react-scheduler-material-ui';
+import TooltipContent from './subcomponents/TooltipContent.js';
+import styles from '../../utils/styles/CalendarStyles/Calendar.module.css';
+
+
+const data = [
+  {
+    title: 'KEvin CLASS',
+    startDate: new Date(2022, 2, 21, 6, 35),
+    endDate: new Date(2022, 2, 21, 22, 30),
+
+    zoomlink: 'http://garbase.com'
+  },
+  {
+  title: 'Other Class',
+  startDate: new Date(2022, 2, 24, 6, 35),
+  endDate: new Date(2022, 2, 24, 10, 30),
+  zoomlink: 'http://assaffsaafs.com'
+},]
+
+
+const PREFIX = 'Pog';
+
+const classes = {
+  todayCell: `${PREFIX}-todayCell`,
+  dayCell: `${PREFIX}-dayCell`,
+  todayHeader: `${PREFIX}-today`,
+  dayHeader: `${PREFIX}-weekday`,
+};
```

Comment:
I suggest meeting with @cmjosephs to go over setting up an import file to handle all of the dynamic data for cleanliness of code. This method allows for the calendar file to only contain the main functions and increases readability.